src/common/Timer.js

```diff
@@ -0,0 +1,20 @@
+class Timer {
```

Comment:
nitpick: not sure if it makes sense here, but should we in general retain copyright of copy/pasted code?